src/com/facebook/buck/testrunner/Junit5TestListener.java

```diff
@@ -179,7 +180,7 @@ public void executionFinished(TestIdentifier testIdentifier, TestExecutionResult
 
     String className = testClass.getCanonicalName();
     String methodName = testIdentifier.getDisplayName().replace("()", "");
-    long runTime = summary.getTimeFinished() - summary.getTimeStarted();
```

Comment:
nit: Should we delete the `summary#getTimeFinished` and `summary#getTimeStarted` if they are now unused ?

Reply:
That's a junit5 class.